frontend/src/pages/LoginView.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Card } from '@components/ant-design/Card';
+import { Button } from '@components/ant-design/Button';
+import { Box } from '@components/atoms/Box';
+import { Paragraph, Title } from '@components/ant-design/Typography';
+import { LoginGraphic, Google } from '@components/svg';
+
+export const LoginView = () => {
+  const click = () => {
```

Comment:
Kiedy tworzysz funkcje obsługujące konkretną akcję (np. kliknięcie) to konwencją jest nazywanie ich `handle[nazwa_akcji]`. W tym przypadku poprawną nazwą będzie `handleClick`